doc/sphinx/source/getting_started/inputdata.rst

```diff
@@ -98,6 +98,10 @@ A list of the datasets for which a cmorizers is available is provided in the fol
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | NSIDC-0116-[nh|sh]           | usi, vsi (day)                                                                                       |   3  | Python          |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
+| OSI-407                      | sic (OImon, day)                                                                                     |   2  | Python          |
```

Comment:
There is no cmorizer for `OSI-407` in thi PR, but for `OSI-409`.

Reply:
Yes, it is a typo. OSI-407 is a drift product, 409 is the sic